src/app/Components/FancyModal/FancyModalHeader.tsx

```diff
@@ -24,6 +24,7 @@ export interface FancyModalHeaderProps {
   useShareButton?: boolean
   rightCloseButton?: boolean
   renderRightButton?: () => JSX.Element
+  alignItems?: "center" | "flex-start" | "flex-end"
```

Comment:
(suggestion): We could use the type from [here](https://github.com/artsy/eigen/blob/e6f452befe0e4c6ec4d37dc474d1df07a4be78f8/src/app/Components/Bidding/Elements/types.d.ts#L129).

```suggestion
  alignItems?: ResponsiveAlignItemsValue
```